internal/operands/metrics/resources.go

```diff
@@ -164,6 +164,22 @@ func getAlertRules() ([]promv1.Rule, error) {
 				componentAlertLabelKey:    componentAlertLabelValue,
 			},
 		},
+		{
+			Alert: "VirtualMachineCRCErrors",
+			Expr:  intstr.FromString("kubevirt_ssp_vm_rbd_volume{volume_mode=\"Block\", rxbounce_enabled=\"false\"} > 0"),
```

Comment:
do you need `or vector(0)` or will this eval just fine?

Reply:
this should be fine, no further actions are taken on the results of `kubevirt_ssp_vm_rbd_volume`